app/assets/stylesheets/splash.scss

```diff
@@ -26,19 +26,16 @@
     .welcome {
       height: 150px;
     }
-    .welcome a {
+    .welcome {
     color: white;
     font-size: 7rem;
-      &:hover {
-        color: darken(white, 20%);
-      }
     }
 
-  .tag_line {
-    padding-top: 30px;
-    font-size: 1.8rem;
-    font-weight: bold;
-  }
+    .tag_line {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Selector `tag_line` should be written in lowercase with hyphens